helm/flux-app/Chart.yaml

```diff
@@ -2,6 +2,7 @@ apiVersion: v2
 appVersion: 0.35.0
 annotations:
   application.giantswarm.io/team: team-honeybadger
+  application.giantswarm.io/internal-upgrade: "true"
```

Comment:
This is to mark the app as subject of internal upgrade. Restrictions would be nicer, maybe, but annotations are easily accessible via helmclient library [as we currently have it](https://github.com/giantswarm/helmclient/pull/426/files). Resigning from it, I would either need to fallback to [Show action](https://pkg.go.dev/helm.sh/helm/v3@v3.10.0/pkg/action#Show) that we currently do not support, or extract the `Chart.yaml` by myself from archive, what I think is not the best idea, or look for restriction in other sources, like metadata, but this would also require extra call for pulling it.